3.9.1/index.d.ts

```diff
@@ -57,8 +23,23 @@ declare interface Gulp {
    * </ul>
    */
   task: typeof Orchestrator.prototype.add;
-  run: RunMethod;
-  watch: typeof watch;
+  /**
+   * Run the specified task(s)
+   * @deprecated will be removed in gulp 4.0. Use task dependencies instead.
+   */
+  run: typeof Orchestrator.prototype.start;
+  /**
+   * Run the specified task(s)
+   * @deprecated will be removed in gulp 4.0. Use task dependencies instead.
+   */
+  start: typeof Orchestrator.prototype.start;
+  /**
+   * watches all files in the specified glob(s) for changes and runs the given task(s).
+   * @param globs
+   * @param tasks
+   */
+  watch(globs?: string | Array<string>, tasks?: task | Array<task>): EventEmitter;
+  watch(globs?: string | Array<string>, opt?: WatchOptions, tasks?: task | Array<task>): EventEmitter;
```

Comment:
@Haringat The VS Code TypeScript IntelliSense is powered by the generic TypeScript language service, just like Atom, Sublime and many others. This is not IDE-specific, that is the official IntelliSense service by TypeScript that is bundled with the compiler on npm. Apparently Webstorm has built its own IntelliSense for TypeScript, but that is not a good reference.
It is simply shorter and also has been used for a long time in JSDoc long before TypeScript existed